controllers/error.controllers.js

```diff
@@ -0,0 +1,7 @@
+const handleCustomErrors = (err, req, res, next) => {
+  if (err.msg && err.status) {
+    res.status(err.status).send({ msg: err.msg });
+  }
```

Comment:
We don't need this error handler yet as it's not being used, so it can be removed until we need it 👍